### PR TITLE
feat(core): wrap github connector

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Wrap Github web connector as a package.

## Testing

UTs.

### CODE REVIEW

File: packages/connector-github/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+  public getAuthorizationUri: GetAuthorizationUri = async (redirectUri, state) => {
+    const config = await this.getConfig(this.metadata.id);
+
+    return `${authorizationEndpoint}?${stringify({
+      client_id: config.clientId,
+      redirect_uri: redirectUri,
+      state,
+      scope, // Only support fixed scope for v1.
+    })}`;
```
Comment (IceHe) [inline packages/connector-github/src/index.ts:52]: seems that it's unnecessary to use `query-string.stringify` and even `query-string`.

e.g.
image

You can refactor it in another PR.

File: packages/connector-github/package.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        "prepack": "pnpm build"
+    },
+    "dependencies": {
+        "@logto/connector-types": "^0.1.0",
+        "@logto/jest-config": "^0.1.0",
+        "@logto/shared": "^0.1.0",
+        "@logto/schemas": "^0.1.0",
+        "@silverhand/essentials": "^1.1.0",
+        "got": "^11.8.2",
+        "query-string": "^7.0.1",
```
Comment (IceHe) [inline packages/connector-github/package.json:32]: ref: https://github.com/logto-io/logto/pull/673/files#r859397225

File: packages/connector-github/src/constant.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  name: {
+    en: 'Sign In with GitHub',
+    'zh-CN': 'GitHub登录',
+  },
+  logo: 'https://user-images.githubusercontent.com/5717882/156983224-7ea0296b-38fa-419d-9515-67e8a9612e09.png',
+  description: {
+    en: 'Sign In with GitHub',
+    'zh-CN': 'GitHub登录',
+  },
+  readme: getFileContents(pathToReadmeFile, readmeContentFallback),
```
Comment (wangsijie) [inline packages/connector-github/src/constant.ts:38]: `__dirname` can be wrapped into `getFileContents`, could you create and issue for that?

**Review by wangsijie — APPROVED** [review]

File: packages/connector-github/src/constant.ts (hunk 3 above)
Comment (darcyYe) [reply packages/connector-github/src/constant.ts:38]: [LOG-2322](https://linear.app/silverhand/issue/LOG-2322/cwd-can-be-wrapped-in-getfilecontents).

**Pull request merged**